dova/www/templates/behavior.html

```diff
@@ -20,6 +20,7 @@
               {{ point.point }}</ion-checkbox>
         </li>
       </ol>
+      
```

Comment:
One little thing about spaces, don't worry too much about them, but when u come by multiple empty lines, try to clean some of them up